src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStripCommands.java

```diff
@@ -20,6 +20,15 @@ public static Command getStaticColorCommand(Color color, LEDStrip... ledStrips)
         );
     }
 
+    public static Command getThreeSectionColorCommand(Color firstSectionColor, Color secondSectionColor, Color thirdSectionColor, LEDStrip... ledStrips) {
```

Comment:
Make this accept suppliers of colors and determine the appropriate color in the LEDAutoSetup class